### DISCUSSION TITLE
백신종류 선택하는 방법이 있나요?

### DISCUSSION THREAD

**Discussion by developer131** [changdoc/naver-vaccine-macro#106] [Q&A]
네이버 예약창으로 들어가서 병원 선택하고 매크로 작동하는 창까지 띄웠는데 백신종류 쪽에 선택해주세요. 라는 메세지가 뜹니다.
특정백신을 선택하여 창을 띄우는 방법이 있는지요
아니면, 그냥 그대로 진행되는 부분인지 궁금합니다.

**Comment by changdoc** [comment]
메시지가 혹시 예약 신청 페이지에 있는

현재 신청가능한 백신이 없습니다
(선착순 소진)

이 문구를 말씀하시는 거라면 잔여 백신이 발생해야 백신 종류와 수량이 목록 형태로 나오게 되고요
본 매크로프로그램은 그 상황일때 잔여 백신 수량이 있는 백신 예약 신청을 자동으로 진행하게됩니다.

**Reply by developer131** [reply]
아~ 감사합니다~!
